code/modules/supply/supply_console.dm

```diff
@@ -275,13 +275,15 @@
 			if(!P.can_order())
 				to_chat(user, "<span class='warning'>That cannot be ordered right now. Please try again later.</span>")
 				return
-
+			if(P.are_you_sure_you_want_to_be_banned)
+				var/we_warned_you = tgui_alert(user, "[P.are_you_sure_you_want_to_be_banned]", "ARE YOU SURE?", list("No", "Yes"))
```

Comment:
```suggestion
				var/we_warned_you = tgui_alert(user, "[P.are_you_sure_you_want_to_be_banned]", "ARE YOU SURE?", list("Yes", "No"))
```
Working on trying to standardized button formatting.

Reply:
fair, was copying the BOH one so people don't instantly click yes

Reply:
The plan is to get them all switched eventually, as we now have a preference option that toggles the button position.

Reply:
Fair enough!